app/src/main/res/drawable/search_widget_favorite_favicon_dark_background.xml

```diff
@@ -15,6 +15,6 @@
   -->
 
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@color/almostBlackDark" />
+    <solid android:color="@color/white12" />
     <corners android:radius="10dp" />
```

Comment:
is this 10dp? can we use an already set radius?

Reply:
There is not any set radius with value `10dp` (S)8dp/(M)12dp/(L)16dp

Reply:
We agreed to change it to 8dp